frontend/apps/reader/modules/readersearch.lua

```diff
@@ -79,6 +80,13 @@ SRELL_ERROR_CODES[110] = _("No preceding expression in repetition.")
 SRELL_ERROR_CODES[111] = _("Expression too complex, some hits will not be shown.")
 SRELL_ERROR_CODES[666] = _("Expression may lead to an extremely long search time.")
 
+function ReaderSearch:registerKeyEvents()
+    if Device:hasKeyboard() then
+        self.key_events.ShowFulltextSearchInput = { { "S" }, event = "ShowFulltextSearchInput", args = "" }
+        self.key_events.ShowFulltextSearchInputRecent = { { "Shift", "S" }, event = "ShowFulltextSearchInput" }
```

Comment:
This is the default option, it should be without `event`.

Reply:
Also it looks like args is reversed.

Reply:
@poire-z wants the event on both so as to make it easier to see that both are doing the same thing

Reply:
> Also it looks like args is reversed.

Do you mean you want S to be the one with the recent one and Shift S the clear field one?

Reply:
If we get to have the 2 events, my preference would be to have them symetric like I proposed.
But I'm fine with whatever if others feels different :)